sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -102,6 +102,12 @@ case class DropDatabaseCommand(
   extends RunnableCommand {
 
   override def run(sparkSession: SparkSession): Seq[Row] = {
+    val catalog = sparkSession.sessionState.catalog
```

Comment:
nit: maybe better to put `catalog `definition into if clause body

Reply:
done